README.md

```diff
@@ -73,6 +73,12 @@ options for the offset in the file/text provided:
 ]
 ```
 
+To use the iOS SDK is to specified the `compilerargs` option. The value
```

Comment:
Then are the quotes wrong in the example? I don't think they are supposed to be there.